src/ort_genai_c.h

```diff
@@ -155,8 +173,9 @@ OGA_EXPORT OgaResult* OGA_API_CALL OgaGeneratorParamsSetInputIDs(OgaGeneratorPar
  */
 OGA_EXPORT OgaResult* OGA_API_CALL OgaGeneratorParamsSetInputSequences(OgaGeneratorParams* generator_params, const OgaSequences* sequences);
 
-OGA_EXPORT OgaResult* OGA_API_CALL OgaGeneratorParamsSetWhisperInputFeatures(OgaGeneratorParams*, const int32_t* inputs, size_t count);
-OGA_EXPORT OgaResult* OGA_API_CALL OgaGeneratorParamsSetWhisperDecoderInputIDs(OgaGeneratorParams*, const int32_t* input_ids, size_t input_ids_count);
+OGA_EXPORT OgaResult* OGA_API_CALL OgaGeneratorParamsAddExtraInput(OgaGeneratorParams* generator_params, const char* name, OgaTensor* tensor);
```

Comment:
Amusingly it was called 'SetInput' initially but that confused people, because it sounds like there's only one and it's special. So it was renamed to 'AddExtraInput' to let you pass in extra data for LoRA models.

Documentation will solve it going forward.